### ISSUE TITLE
Stretch Goal Planning

### ISSUE THREAD

**Issue by fmings** [fmings/INDIVIDUAL-ASSIGNMENT-Team-Roster#14]
Create an MVP branch from the main branch after completing all MVP and push it up to Github before starting stretch goals
Take time to plan and create tickets for Stretch Goals

**Comment by fmings** [comment]
Stretch Goals 1 & 2 planned.
Will created separate ticket for stretch goals 3 & 4
